src/test/java/seedu/cakecollate/testutil/TypicalOrders.java

```diff
@@ -32,39 +32,48 @@
     public static final Order ALICE = new OrderBuilder().withName("Alice Pauline")
             .withAddress("123, Jurong West Ave 6, #08-111").withEmail("alice@example.com")
             .withPhone("94351253").withOrderDescriptions("2 x Strawberry Cakes").withTags("friends")
-            .withDeliveryDate("01/01/2022").withDeliveryStatus().build();
+            .withDeliveryDate("01/01/2022").withDeliveryStatus()
+            .withRequest("Give me more pineapples.").build();
     public static final Order BENSON = new OrderBuilder().withName("Benson Meier")
             .withAddress("311, Clementi Ave 2, #02-25")
             .withEmail("johnd@example.com").withPhone("98765432")
             .withOrderDescriptions("2 x Chocolate Cakes")
             .withTags("owesMoney", "friends")
-            .withDeliveryDate("01-01-2022").withDeliveryStatus().build();
+            .withDeliveryDate("01-01-2022").withDeliveryStatus()
+            .withRequest("More pineapples").build();
     public static final Order CARL = new OrderBuilder().withName("Carl Kurz").withPhone("95352563")
             .withEmail("heinz@example.com").withAddress("wall street").withOrderDescriptions("1 Vanilla Cake")
-            .withDeliveryDate("01.01.2022").withDeliveryStatus().build();
+            .withDeliveryDate("01.01.2022").withDeliveryStatus()
+            .withRequest("").build();
     public static final Order DANIEL = new OrderBuilder().withName("Daniel Meier").withPhone("87652533")
             .withEmail("cornelia@example.com").withDeliveryStatus().withAddress("10th street")
             .withOrderDescriptions("1 Chocolate Chip Muffin").withTags("friends")
-            .withDeliveryDate("01 Jan 2022").withDeliveryStatus().build();
+            .withDeliveryDate("01 Jan 2022").withDeliveryStatus()
+            .withRequest("").build();
     public static final Order ELLE = new OrderBuilder().withName("Elle Meyer").withPhone("9482224")
             .withEmail("werner@example.com").withAddress("michegan ave")
             .withOrderDescriptions("1 Raisins Cake Thing", "1 x Blackforest cake", "1 x Vanilla Cake")
-            .withDeliveryDate("31/12/2022").withDeliveryStatus().build();
+            .withDeliveryDate("31/12/2022").withDeliveryStatus()
+            .withRequest("").build();
     public static final Order FIONA = new OrderBuilder().withName("Fiona Kunz").withPhone("9482427")
```

Comment:
Think a variety of requests will be good here too

Reply:
added more requests